recipe/meta.yaml

```
@@ -1,12 +1,12 @@
 # Make sure the component versions are consistent with the SDK version!
-{% set version = "23.06.1.8" %}
-{% set cusv_version = "1.4.1" %}
-{% set cutn_version = "2.2.1" %}
+{% set version = "23.10.0" %}
```

The reason we need 4 numbers here was because only this could make the CF bot work, but TBH I dislike it. Once the bot is decommissioned we should remove the 4th digit.   
```suggestion
{% set version = "23.10.0.6" %}
```

---

Why "6"? Any particular reason?

---

It's used to compute the tarball url, and listed in the json file that we discussed:
https://developer.download.nvidia.com/compute/cuquantum/redist/redistrib_23.10.0.json

---

I think in this case it's coming from the build number set in Jeremy's pipelines.